Source/Classes/PINRemoteImageDownloadQueue.h

```diff
@@ -18,7 +18,7 @@ typedef void (^PINRemoteImageDownloadCompletion)(NSURLResponse * _Nullable respo
 
 @interface PINRemoteImageDownloadQueue : NSObject
 
-@property (nonatomic, assign) NSUInteger maxNumberOfConcurrentDownloads;
+@property (atomic, assign) NSUInteger maxNumberOfConcurrentDownloads;
```

Comment:
This property is actually thread-safe so I thought it'd be a good idea to let clients know that.